mentors/javascript/s.json

```diff
@@ -56,4 +56,11 @@
     "avatar_url": null,
     "bio": "JavaScript has been part of my day-to-day job for over 7 years -- A love-hate relationship"
   }
+  {
+    "github_username": "Hakeem-Olasupo",
```

Comment:
Your username begins with the latter `H`.  Your bio information should be entered into `h.json` and not `s.json`.

Reply:
@digitalforus Could you update the PR according to the comment made by @rpottsoh? Otherwise we won't be able to merge this.